pkgs/by-name/do/dopewars/package.nix

```diff
@@ -0,0 +1,39 @@
+{ lib, stdenv, fetchFromGitHub, makeWrapper,
+  autoconf, automake, curl, ncurses, gtk3, pkg-config,
+  scoreDirectory ? "$HOME/.local/share" }:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "dopewars";
+  version = "1.6.2";
+
+  src = fetchFromGitHub {
+    owner = "benmwebb";
+    repo = "dopewars";
+    rev = "v${finalAttrs.version}";
+    sha256 = "sha256-CpgqRYmrfOFxhC7yAS2OqRBi4r3Vesq3+7a0q5rc3vM=";
+  };
+
+  nativeBuildInputs = [ pkg-config autoconf automake makeWrapper ];
+  buildInputs = [ curl gtk3 ncurses ];
+
+  # remove the denied setting of setuid bit permission
+  patches = [ ./0001-remove_setuid.patch ];
+
+  preConfigure = ''
```

Comment:
Have you tried `autoreconfHook` instead of calling autogen.sh?

Reply:
I haven't tried  `autoreconfHook` before. I'll implement it right away. Thanks for your help!